src/App.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
+import { PokerCard } from "./components";
 import "./styles.css";
 
 export default function App() {
   return (
     <div className="App">
-      <h1>Welcome to your second mini challenge!</h1>
+      <PokerCard />
```

Comment:
It would be better to inject mock data here instead of inside the component.
Advantages:
 - Unit testing
 - Usage of the component with no repeating your self

Reply:
I have worked with React Testing Library but the thing here was I had some troubles using it with Codesanbox (Never used before).